### ISSUE TITLE
Keyboard shortcuts for tools

### ISSUE THREAD

**Issue by ezhang816** [OrionMD/pacsbin-community#25]
Hi Jason,

It would be nice to have customizable keyboard shortcuts for every tool.

Thanks!

**Comment by jmhmd** [comment]
I agree! Will add to the list. This can get tricky, trying to avoid collisions with other standard webpage shortcuts, but would be nice with some limits.
